pontoon/base/models/user.py

```diff
@@ -7,11 +7,15 @@
 from django.conf import settings
 from django.contrib.auth.models import User
 from django.contrib.contenttypes.models import ContentType
-from django.db.models import Count
+from django.db.models import Count, Q
 from django.urls import reverse
 from django.utils import timezone
 
 from pontoon.actionlog.models import ActionLog
+from pontoon.base.utils import aware_datetime
+
+
+BADGE_START_DATE = aware_datetime(2024, 10, 18)
```

Comment:
We should move this variable to settings, so we don't need to deploy new code to change the date. The default value could be None and the code should be adjusted accordingly.